relentless/simulate/lammps.py

```diff
@@ -304,6 +304,7 @@ def pair_map(sim,pair):
         if sim.communicator.rank == sim.communicator.root:
             with open(file_,'w') as fw:
                 fw.write('# LAMMPS tabulated pair potentials\n')
+                rcut = {}
```

Comment:
This variable needs to be made available on all ranks, right now it is only defined on the root rank. To do this, you either need to broadcast from `root`, or you need to adapt the code so that `u` and `f` become available on all ranks. The first option requires fewer changes:
```py
if sim.communicator.rank == sim.communicator.root:
    rcut = {}
    with open(file_,'w') as fw:
        # all the existing code here
else:
    rcut = None
rcut = sim.communicator.bcast(rcut)
```